packages/edge-login-ui-rn/src/common/locales/en_US.js

```diff
@@ -154,7 +154,9 @@ const strings = {
   four_digit_pin_error: 'PIN must be four digits',
   it_would_take_xx_to_crack: 'It would take %s to crack your password',
   recommend_choosing_a_stronger: '\nWe recommend choosing a stronger password',
-  backup_key_incorrect: 'Backup Key was incorrect'
+  backup_key_incorrect: 'Backup Key was incorrect',
+  something_went_wrong: 'Something Went Wrong',
```

Comment:
@paullinator OK, so this is the title of the dialog box. 
The error message from the code is actually being displayed as the error message. So I can make the message body of the dialog be the wording you are suggesting. What should the title of the dialog box be?

Reply:
How about this, let's concat the error messages.

So it's...

Title: Error occurred creating account

"An error occurred creating your account. This may have been due to a slow network connection or network interruption. Please retry and if you receive another error that the account already exists, please restart the app and login with your username and password."

Error: [Error message from core]

Reply:
Your changes still have "Something went wrong" as the title instead of "Error occurred creating account" and you didn't concat the error from core at the bottom.

Reply:
@paullinator Perfect meant I was agreeing with you .. It is complete now.